engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingstd/ByteRollingStdOperator.java

```diff
@@ -22,6 +22,7 @@
 
 public class ByteRollingStdOperator extends BaseDoubleUpdateByOperator {
     private static final int BUFFER_INITIAL_CAPACITY = 128;
+    private static final double VARIANCE_THRESHOLD = 1e-12;
```

Comment:
As presented, this change can not go in.  For example, consider a physics group that is using DH to measure the mass of an electron (9.11E-28 g), which I will call m_e.  The value of m_e is >> 2E-308.  As a result, it is very reasonable to compute the variance of m_e measurements and expect good numerics.  By choosing a hard limit of 1E-12, the m_e experiment always returns a variance of 0.0, which is BS.

It is very reasonable to put a hard limit of 0.0, but putting a non-zero constant threshold is not viable.

Numpy's impl is here.  I have not dug through all of the details, but you can see that there is no hard limit.
https://github.com/numpy/numpy/blob/v1.25.0/numpy/core/fromnumeric.py#L3654-L3788
https://github.com/numpy/numpy/blob/v1.25.0/numpy/core/_methods.py#L135

We may be able to look at the constituent values to determine something useful.  e.g. if all the values are the same, the variance is zero.

Reply:
Implemented the floor of `0.0` to cover only the negative variance case.  This should eliminate the `NaN` while still giving accurate results (to the limits of floating point precision).